packages/docs/src/components/layout/Layout.tsx

```diff
@@ -94,23 +94,25 @@ const Layout = ({
           className="ds-l-md-col ds-u-padding--0 ds-u-padding-bottom--4 ds-u-padding-top--2 page-main"
         >
           {pageHeader ? pageHeader : <PageHeader frontmatter={frontmatter} theme={theme} />}
-          <article className="ds-u-md-display--flex ds-u-padding-x--3 ds-u-sm-padding-x--6 ds-u-sm-padding-bottom--6 ds-u-sm-padding-top--1 ds-u-padding-bottom--3 page-content">
-            <div className="page-content__content ds-l-lg-col--9 ds-u-padding-left--0">
-              <div className="ds-u-display--block ds-u-lg-display--none">
-                <TableOfContentsMobile
+          <article className="ds-u-md-display--flex ds-u-padding-x--3 ds-u-sm-padding-x--6 ds-u-sm-padding-bottom--6 ds-u-sm-padding-top--1 ds-u-padding-bottom--3 page-body">
+            <div className="ds-l-row">
```

Comment:
The big change here that is hard to see is that I've wrapped these two columns (which is only one column on small screens) in a `ds-l-row`, which allows us to remove a special rule that zeroed out column paddings when the screen got to a certain width (which was an indirect way of determining if two columns were visible).